package.json

```diff
@@ -122,13 +122,19 @@
     "lint": "xo",
     "pretest": "npm run lint",
     "semantic-release": "./bin/semantic-release.js",
-    "test": "nyc ava -v"
+    "test": "nyc ava -v",
+    "test:ci": "nyc ava -v"
```

Comment:
is it worth having this as a separate script when it already matches the existing `test` script? i know we needed it, at least, temporarily in some cases where they didnt match, but seems like that could be avoided here.

Reply:
I added it because of `pretest`. On CI, we run the linting in the separate job, that only runs after the tests passed in all supported node versions